src/components/chats/components/CreateNewChat/CreateStandaloneChat.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from '@chakra-ui/react'
+import { DocumentIcon } from '@heroicons/react/24/outline'
+import { FC } from 'react'
+import { useCreateChat } from '../../chatHooks'
+import { NewChatProps } from './types'
+import { useCanCreateChat } from './useCanCreateChat'
+
+export const CreateStandaloneChat: FC<NewChatProps> = ({
+  postId,
+  chatId,
+  resetTextArea,
+}) => {
+  const { mutate: createChat } = useCreateChat()
```

Comment:
Now that we have 2 hooks, shall we rename to `useCreateChatbotChat` ?

Reply:
I was keeping consistency with names but maybe we can define the new standard